aptos/contracts/sources/price_feed.move

```diff
@@ -0,0 +1,37 @@
+module pyth::price_feed {
+    use pyth::price_identifier::PriceIdentifier;
+    use pyth::price::Price;
+
+    /// PriceFeed represents a current aggregate price for a particular product.
+    struct PriceFeed has copy, drop, store {
+        /// The price identifier
+        price_identifier: PriceIdentifier,
```

Comment:
`id`? I don't have strong opinion here though.

Reply:
suggest calling the field `id` and the type `PriceIdentifier`